agent/06_agent_create_cluster.sh

```diff
@@ -23,9 +23,55 @@ function create_pxe_files() {
 function create_image() {
     local asset_dir=${1}
     local openshift_install=${2}
+
+    if [ "${AGENT_USE_APPLIANCE_MODEL}" == true ]; then
+      create_factory_image
+    else
+      create_automated_image
+    fi
+}
+
+function create_automated_image() {
     "${openshift_install}" --dir="${asset_dir}" --log-level=debug agent create image
 }
 
+function create_factory_image() {
+    config_image_drive="sdd"
+
+    # The command to create the config-image must be run out of a separate asset directory using same assets
+    mkdir -p ${config_image_dir}
+    cp ${asset_dir}/*.yaml ${config_image_dir}
+
+    # Create the unconfigured ingition and include it in an ISO
+    "${openshift_install}" --dir="${asset_dir}" --log-level=debug agent create cluster-manifests
```

Comment:
After the "create cluster-manifests" line, could we add these two lines?
```
   rm "${asset_dir}/cluster-manifests/nmstateconfig.yaml"
   rm "${asset_dir}/.openshift_install_state.json"
```
This will ensure that the nmstateconfig is not in the unconfigured-ignition. Would also make break ssh and the network until the config image contents is copied and applied to the host. If the cluster installation succeeds then we know that the static network config has been applied correctly by load-config-iso.sh in https://github.com/openshift/installer/pull/7200.

Reply:
Yes, good suggestion, I will make that change. Note that until the config-image is applied it is possible to ssh to 192.168.11.20 - the address picked up via DHCP.